In-house patch to remove unnecessary dependencies from Cinder's
requirements files. The specific reasons are as follows:

anyjson		Not applicable

kombu		Not applicable

oslo.rootwrap	Not applicable to Solaris

pycrypto	Not applicable to Solaris (various drivers specific)

rtslib-fb	Not applicable to Solaris (Linux iSCSI specific)

--- cinder-2015.1.2/cinder.egg-info/requires.txt.~1~	2015-10-13 09:30:55.000000000 -0700
+++ cinder-2015.1.2/cinder.egg-info/requires.txt	2016-01-30 21:00:52.192406009 -0800
@@ -1,11 +1,9 @@
 pbr!=0.7,<1.0,>=0.6
-anyjson>=0.3.3
 Babel>=1.3
 eventlet!=0.17.0,>=0.16.1
 greenlet>=0.3.2
 iso8601>=0.1.9
 keystonemiddleware<1.6.0,>=1.5.0
-kombu>=2.5.0
 lxml>=2.3
 netaddr>=0.7.12
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
@@ -15,14 +13,12 @@ oslo.db<1.8.0,>=1.7.0 # Apache-2.0
 oslo.log<1.1.0,>=1.0.0 # Apache-2.0
 oslo.messaging<1.9.0,>=1.8.0 # Apache-2.0
 oslo.middleware<1.1.0,>=1.0.0 # Apache-2.0
-oslo.rootwrap<1.7.0,>=1.6.0 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
 oslo.utils!=1.4.1,<1.5.0,>=1.4.0 # Apache-2.0
 osprofiler>=0.3.0 # Apache-2.0
 paramiko>=1.13.0
 Paste
 PasteDeploy>=1.5.0
-pycrypto>=2.6
 pyparsing>=2.0.1
 python-barbicanclient<3.1.0,>=3.0.1
 python-glanceclient<0.18.0,>=0.15.0
@@ -32,7 +28,6 @@ requests!=2.4.0,>=2.2.0
 retrying!=1.3.0,>=1.2.3 # Apache-2.0
 Routes!=2.0,>=1.12.3
 taskflow<0.8.0,>=0.7.1
-rtslib-fb>=2.1.41
 six>=1.9.0
 SQLAlchemy<=0.9.99,>=0.9.7
 sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
--- cinder-2015.1.2/requirements.txt.~1~	2015-10-13 09:27:35.000000000 -0700
+++ cinder-2015.1.2/requirements.txt	2016-01-30 21:01:27.270741292 -0800
@@ -3,13 +3,11 @@
 # process, which may cause wedges in the gate later.
 
 pbr!=0.7,<1.0,>=0.6
-anyjson>=0.3.3
 Babel>=1.3
 eventlet!=0.17.0,>=0.16.1
 greenlet>=0.3.2
 iso8601>=0.1.9
 keystonemiddleware<1.6.0,>=1.5.0
-kombu>=2.5.0
 lxml>=2.3
 netaddr>=0.7.12
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
@@ -19,14 +17,12 @@ oslo.db<1.8.0,>=1.7.0 # Apache-2.0
 oslo.log<1.1.0,>=1.0.0 # Apache-2.0
 oslo.messaging<1.9.0,>=1.8.0 # Apache-2.0
 oslo.middleware<1.1.0,>=1.0.0 # Apache-2.0
-oslo.rootwrap<1.7.0,>=1.6.0 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
 oslo.utils!=1.4.1,<1.5.0,>=1.4.0 # Apache-2.0
 osprofiler>=0.3.0 # Apache-2.0
 paramiko>=1.13.0
 Paste
 PasteDeploy>=1.5.0
-pycrypto>=2.6
 pyparsing>=2.0.1
 python-barbicanclient<3.1.0,>=3.0.1
 python-glanceclient<0.18.0,>=0.15.0
@@ -36,7 +32,6 @@ requests!=2.4.0,>=2.2.0
 retrying!=1.3.0,>=1.2.3 # Apache-2.0
 Routes!=2.0,>=1.12.3
 taskflow<0.8.0,>=0.7.1
-rtslib-fb>=2.1.41
 six>=1.9.0
 SQLAlchemy<=0.9.99,>=0.9.7
 sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
